api/XMLDocument.json

```
@@ -22,24 +38,71 @@
           "ie": {
             "version_added": false
           },
-          "opera": {
-            "version_added": false
-          },
-          "opera_android": {
-            "version_added": false
-          },
-          "safari": {
-            "version_added": false
-          },
-          "safari_ios": {
-            "version_added": false
-          },
-          "samsunginternet_android": {
-            "version_added": "5.0"
-          },
-          "webview_android": {
-            "version_added": "45"
-          }
+          "opera": [
+            {
+              "version_added": "21"
+            },
+            {
+              "version_added": true,
+              "version_removed": "21",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "opera_android": [
+            {
+              "version_added": "21"
+            },
+            {
+              "version_added": true,
+              "version_removed": "21",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "safari": [
+            {
+              "version_added": "10"
+            },
+            {
+              "version_added": "3",
+              "version_removed": "10",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": "10"
+            },
+            {
+              "version_added": "1",
+              "version_removed": "10",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "samsunginternet_android": [
+            {
+              "version_added": "2.0"
+            },
+            {
+              "version_added": "1.0",
+              "version_removed": "2.0",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "webview_android": [
+            {
+              "version_added": "37"
+            },
+            {
+              "version_added": "≤37",
```

I was just taking a look at this again, and I've just realized that we can actually place a real version number here.  It was mentioned somewhere by Joe that it's safe to assume that all features implemented in Chrome 1 were also present in WebView for Android 1.  Let's set it here, and then we no longer have to worry about the linter complaining for this PR:

```suggestion
              "version_added": "1",
              "version_removed": "37",
```